apps/src/maze/api.js

```diff
@@ -158,143 +160,142 @@ function moveAbsoluteDirection(direction, id) {
   Maze.executionInfo.stopCollecting();
 }
 
-exports.moveForward = API_FUNCTION(function (id) {
+export var moveForward = API_FUNCTION(function (id) {
```

Comment:
Why `var` rather than `const`?

Reply:
I used lebab for this, and the commonjs conversion does `var` rather than `const` by default, unless I also want to update all other instances of `var` to `let` or `const`. Which I do, but just in another PR